src/assets/constants.ts

```diff
@@ -1,19 +1,16 @@
-import LocalePaths from "../types/constants";
+import LocalePaths from '../types/constants';
 
 export const LOCALE_PATHS: LocalePaths = {
-	login: '/login',
-	auth: '/auth',
-	main: '/main',
-	empty: '/',
+	messenger: '/messenger',
+	signUp: '/sign-up',
+	settings: '/settings',
+	notfound: '/notfound',
+	main: '/',
```

Comment:
Я бы предложил заменить на union type
Чем меньше кода в рантайме, тем лучше, потому что тем меньше ошибок можно в рантайме выхватить.
TypeScript будет автокомплитом подсказывать какое значение вставить
```suggestion
type LocalePaths =
  | '/messenger
  | '/auth'
  | '/main'
  | '/'
```